src/scrabble.js

```diff
@@ -1,5 +1,34 @@
-function scrabble() {
+const points = require('./points')
```

Comment:
Like that you have created separate files to call upon the infomation you need so as to limit clutter in your code